AwsCryptographicMaterialProviders/runtimes/rust/src/dafny_libraries.rs

```diff
@@ -130,20 +130,42 @@ pub mod DafnyLibraries {
             }
         }
 
+        pub fn INTERNAL_AppendBytesToFile(
```

Comment:
If this is an internal function, can we make it not `pub`?

Reply:
Same for `INTERNAL_WriteBytesToFile`

Reply:
The INTERNAL things need to be pub(crate) because they're the actual externs.
Other things can be fully private.